Providers/Scripts/2.4x-2.5x/Scripts/nxOMSCustomLogNew.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved. 
+#============================================================================
+
+import os
+import imp
+import re
+import codecs
+import shutil
+import string
+import filecmp
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+CONF_PATH = '/etc/opt/microsoft/omsagent/conf/omsagent.d/customlog.conf'
+RESOURCE_DIR = '/opt/microsoft/omsconfig/modules/nxOMSCustomLogNew/DSCResources/MSFT_nxOMSCustomLogNewResource/'
+RESOURCE_PLUGIN_DIR = RESOURCE_DIR + 'CustomLog/Plugin/'
+PLUGIN = RESOURCE_PLUGIN_DIR + 'in_sudo_tail.rb'
+SCRIPT = RESOURCE_PLUGIN_DIR + 'tailfilereader.rb'
+AGENT_PLUGIN_DIR = '/opt/microsoft/omsagent/plugin/'
+AGENT_PLUGIN = AGENT_PLUGIN_DIR + 'in_sudo_tail.rb'
+AGENT_SCRIPT = AGENT_PLUGIN_DIR + 'tailfilereader.rb'
+
+'''
+[ClassVersion("1.0.0")]
+class MSFT_nxOMSCustomLogNewObject
+{
+    [key] string LogName;
+    [write] string FilePath[];
+};
+
+[ClassVersion("1.0.0")]
+class MSFT_nxOMSCustomLogNewResource : OMI_BaseResource
+{
+    [key] string Name;
+    [write] boolean EnableCustomLogConfiguration;
+    [write,ValueMap{"Present", "Absent"},Values{"Present", "Absent"}] string Ensure;
+    [write, EmbeddedInstance("MSFT_nxOMSCustomLogNewObject") : ToSubclass] string CustomLogObjects[];
+};
+'''
```

Comment:
Just out of curiosity, why did you include this multi-line python comment giving the schema MOF? It can be referenced from the actual file in this PR.